File: oka-web/src/app/Movement.ts
export class Movement {

  dice: number;
  end: boolean;
  turn: number;
  nextTurn: number;
  from: number;
  to: number;
  jumpInfo: Movement;

  winner: number;
  nitems: number;

  hasJump() {
    return this.jumpInfo;
  }
}

File: api/src/main/java/es/fpg/oka/model/oka/Movement.java
package es.fpg.oka.model.oka;

import lombok.Data;

@Data
public class Movement {

	private int dice;
	private boolean end;
	private int turn;
	private int nextTurn;
	private int from;
	private int to;
	private int nitems;
	private Jump jumpInfo;
	private GameStatus status;
	
	@Data
	public static class Jump {
		
		private int from;
		private int to;
	}
	
	public boolean isJump() {
		return jumpInfo != null;
	}
}
